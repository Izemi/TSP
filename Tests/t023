#!/bin/bash
# Two airports at same location

trap "/usr/bin/killall -q -u $USER ./TSP 2>/dev/null" 0 1 2 3 9 15
trap "/bin/rm -f $STDERR" 0 1 2 3 9 15
if [ ! -x ./TSP ]; then
  echo './TSP is missing or not executable'
  echo './TSP is missing or not executable' 1>&2
  exit 1
fi


$BIN/run -stderr=stderr.out ./TSP $WHERE/t023.in -insert HVN 00P 01A < /dev/null
cat stderr.out
