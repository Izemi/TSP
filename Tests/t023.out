-insert   :     11381.33 HVN 00P 01A HVN
